documentation/docs/80-adapter-api.md

```diff
@@ -23,15 +23,17 @@ export default function (options) {
 
 The types for `Adapter` and its parameters are available in [types/config.d.ts](https://github.com/sveltejs/kit/blob/master/packages/kit/types/config.d.ts).
 
-Within the `adapt` method, there are a number of things that an adapter should do:
+Within the `adapt` method, an adapter should
 
 - Clear out the build directory
-- Output code that:
```

Comment:
I liked having this line because it separates what the adapter itself is doing vs what is being done by the code that it outputs

Reply:
I agree the separation is nice, but it looks awkward to have nested lists in the site. I don't remember Sapper or Svelte docs with this types of list, not visibly at least, and the styles doesn't look quite right. We can probably achieve better separation with headings and/or numbered list.